### DISCUSSION TITLE
Web page - Execute javascript or manupulate DOM

### DISCUSSION THREAD

**Discussion by Zack0ne** [rocksdanister/lively#1653] [Q&A] [answered]
Is it possible to execute javascript or manupulate DOM of Web page wallpaper after it loaded?
For example [Genshin Impact animated background with UI elements](https://act.hoyoverse.com/ys/event/e20221028nahida/index.html)
`document.getElementById("frame").hidden = true` should hide the UI.

**Comment by rocksdanister** [comment, marked as answer]
Should be possible by enabling devtools.

Settings -> Wallpapers -> Web browser -> Debugging port

**Reply by Zack0ne** [reply]
I mean does it possible execute it automatically when applying the wallpaper. (on startup, boot .etc). it's CORS protected. so self serving as iframe does not seem to work. Sorry I should've specify that

**Reply by rocksdanister** [reply]
Not possible for users to do right now.

**Reply by Zack0ne** [reply]
Oh ok. Thank you very much
